--- /mnt/data/badextract/iphone5dualbooter/assets/build_patched_idevicerestore.sh	2026-07-20 03:20:59.000000000 +0000
+++ iphone5dualbooter/assets/build_patched_idevicerestore.sh	2026-07-20 03:31:03.383377830 +0000
@@ -403,7 +403,7 @@
     '-L/usr/local/lib -L/usr/lib/x86_64-linux-gnu -Wl,-rpath,/usr/local/lib -lzstd -llzma -lbz2',
 )
 
-libimobile_stage = text.find('echo "Building libimobiledevice...")
+libimobile_stage = text.find('echo \"Building libimobiledevice...\"')
 if libimobile_stage < 0:
     raise SystemExit("Could not locate compile.sh libimobiledevice stage")
 libimobile_autogen = text.find('./autogen.sh', libimobile_stage)
@@ -416,7 +416,7 @@
 if transport_command not in text:
     text = text[:libimobile_autogen] + transport_command + text[libimobile_autogen:]
 
-stage = text.find('echo "Building libirecovery...")
+stage = text.find('echo \"Building libirecovery...\"')
 autogen = text.find('./autogen.sh $CONF_ARGS $CC_ARGS', stage)
 if stage < 0 or autogen < 0:
     raise SystemExit("Could not locate compile.sh libirecovery stage")
